lingua_franca/format.py

```diff
@@ -392,6 +429,34 @@ def nice_year(dt, lang='', bc=False):
     return date_time_format.year_format(dt, full_code, bc)
 
 
+@localized_function(run_own_code_on=[FunctionNotLocalizedError])
+def get_date_strings(dt=None, date_format='MDY', time_format="full", lang=""):
+    lang = get_primary_lang_code(lang)
+    dt = dt or now_local()
+    timestr = nice_time(dt, lang, speech=False,
+                         use_24hour=time_format == "full")
+    monthstr = nice_month(dt, date_format, lang)
+    weekdaystr = nice_weekday(dt, lang)
+    yearstr = dt.strftime("%Y")
+    daystr = nice_day(dt, date_format, include_month=False, lang=lang)
+    if date_format == 'MDY':
+        dtstr = dt.strftime("%-m/%-d/%Y")
+    elif date_format == 'DMY':
+        dtstr = dt.strftime("%d/%-m/%-Y")
+    elif date_format == 'YMD':
+        dtstr = dt.strftime("%Y/%-m/%-d")
+    else:
+        raise ValueError("invalid date_format")
+    return {
+        "date_string": dtstr,
```

Comment:
I feel we should include abbreviations coming from
`nice_month`/`nice_weekday`with a changed signature
```python
def nice_month(dt, abbreviation=False, lang=""):
```
in both cases date_format is irrelevant